README.md

```diff
@@ -180,6 +182,78 @@ To build the documentation run `npm run docs` and the documentation will be acce
 
 **[View Strider API Docs](http://strider-api-docs.surge.sh/)**
 
+## Backers
+
+Support us with a monthly donation and help us continue our activities. [[Become a backer](https://opencollective.com/strider#backer)]
+
+<a href="https://opencollective.com/strider/backer/0/website" target="_blank"><img src="https://opencollective.com/strider/backer/0/avatar.svg"></a>
```

Comment:
They all resolve into 1px transparent png if there is no backer to show (so it doesn't pollute visually for the end user, it's just annoying in the markdown version). It's sadly the only way we can make each backer link to their own profile.